### PR TITLE
Milestone 6 - documentation

### PR DESCRIPTION
**Pull request by aboongm**
This is an Online Web Development Courses Website. 

This website currently has two pages viz home page and about page. The home page has different sections arranged in an intuitive manner.  The sections on the home page are navigation, main, program, instructors, partners, and footer section.

The about page has navigation, main, logo story, past programs, partners, and footer sections.

The main interesting feature of this website is it's responsive and changes the content of the website at the 768 pixels breakpoint. The browser window width less than 768 pixels is for mobile screens and the browser window width greater than 768 pixels is for the desktop screen. We can look at these features by resizing the browser window.

There are sections on the page that disappear or reappear depending on whether its a mobile or desktop screen.

Loom Video Link on the project:
https://www.loom.com/share/086da32123974a9b81d2d4ba08932616

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,6 @@
   </head>
   <body>
     <section class="container bg-light p-0 m-0">
-      <!-- top panel -->
       <header class="m-0 p-0 container">
```
Comment (thecodechaser) [inline index.html:15]: - [ ] Good job on the design but for this project you are supposed to stick to the [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template as much as possible.
  - Please place more gap between your menu items and make sure it matches the design on the template.
  - Please remove the background color from your campaign button and place the border as on the template.

### Your design

image

## Behnace design

image

File: index.html [hunk 2]
```diff
@@ -62,12 +59,11 @@
...
-            <div class="locate fs-5">@Nations of the World, Allow your citizens to learn Web Development!</div>
+            <div class="locate fs-5">@Starting this date, come and join to learn Web Development!</div>
           </div>
         </section>
       </section>
 
-      <!-- program -->
       <section id="program" class="program-bg">
         <div class="container-md p-0 my-0 mx-auto">
           <h3 class="cocogoose text-white text-center h4 fw-bold pt-5">Main Program</h3>
```
Comment (thecodechaser) [inline index.html]: - [x] When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hove mouse over them.

### See the image for reference 

image

Comment (thecodechaser) [inline index.html]: - [ ] When I see the page on the mobile screen the design is appearing differently for the cards, Please make sure it matches the design on the template for all screens. You may want to reduce the font size.

### Your design

image

### Behance design

image

File: index.html [hunk 3]
```diff
@@ -164,7 +159,6 @@ <h3 class="cocogoose text-dark text-center h4 fw-bold pt-5">Featured Instructors
         </div>
       </section>
 
-      <!-- partners -->
       <section id="partner" class="partner-bg toggle-1">
         <div class="container-md p-3">
           <h3 class="text-partner text-center h3">Partners</h3>
```
Comment (thecodechaser) [inline index.html]: - [ ] Please increase the size of the partner's image and make sure it matches the design on the template.

### Your design

image

### Behance design

image

File: index.html [hunk 4]
```diff
@@ -12,7 +12,6 @@
   </head>
   <body>
     <section class="container bg-light p-0 m-0">
```
Comment (thecodechaser) [inline index.html:14]: - [ ] When I open the mobile menu and switch to the desktop screen the mobile menu is still on the screen it should disappear. Please make sure when we switch the screen from mobile to desktop the mobile menu should disappear.
 - You can use media queries for that if the screen cross the min-width you can set your mobile menu `display: none`

### See the image for reference 

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🔴🟡 🔴 

## Hi @aboongm 👋 

### You are doing a marvelous job on the capstone project and this is one of the best projects I have seen so far!
I really like your presentation for the project 👍

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Good commit messages. ✔️

## Required Changes ♻️
- [ ]  In the presentation video you should show some piece of code that you are very proud of, Please make sure your presentation video includes that. (Please also include the presentation video in the PR)
- [ ] Please include project features in more detail in the pull request summary.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @aboongm 🙋‍♀️, 
Good job in making the required changes 👏 
Excellent work on the project.

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**